### PR TITLE
Ret 1464 - Update data model

### PR DESCRIPTION
**Pull request by lvo700**
https://tools.hmcts.net/jira/browse/RET-1464

Updated et-data-model in build.gradle.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: src/main/resources/application.yaml [hunk 1]
```diff
@@ -34,7 +34,7 @@ idam:
   s2s-auth:
     url: ${SERVICE_AUTH_PROVIDER_URL:http://localhost:4502}
     secret: ${ET_SYA_S2S_SECRET:AAAAAAAAAAAAAAAC}
-    microservice: ${MICRO_SERVICE:et_sya_api}
+    microservice: ${MICRO_SERVICE:ccd_gw}
```
Comment (jthmcts) [inline src/main/resources/application.yaml:37]: Are you sure about this change?

Comment (marcjudge) [inline src/main/resources/application.yaml:37]: Just return this to et_sya_api for now. I know it needs to be like the above to work locally but we need to update ecm-ccd-docker to fix that.

**Review by marcjudge — APPROVED** [review]

File: src/main/resources/application.yaml (hunk 1 above)
Comment (lvo700) [reply src/main/resources/application.yaml]: Changed back to et_sya_api

Comment (lvo700) [reply src/main/resources/application.yaml]: Changed back to et_sya_api

**Review by jthmcts — APPROVED** [review]

**Review by marcjudge — APPROVED** [review]

**Review by jsinghm13 — APPROVED** [review]

**Pull request merged**